### PR TITLE
gui: Use SDL_OpenURL in search

### PR DESCRIPTION
**Pull request by nishinji**
Macdu's advice helped me fix it.
Japanese and other languages(like chinese) can be handled.

### CODE REVIEW

**Comment by Zangetsu38** [comment]
Thx, i have never things how fix this

**Comment by Zangetsu38** [comment]
@nishinji already yiu need run format, probably order for header added

**Comment by nishinji** [comment]
@Zangetsu38 fixed

**Comment by Macdu** [comment]
Wouldn't it be better for open_path to use SDL_OpenUrl if the path starts with http ?

**Comment by nishinji** [comment]
Is it preferable to replace open_path with sdl_openurl or rewrite the contents of open_path?

**Comment by Macdu** [comment]
I would say rewrite the content of open_path so that if it starts with http, use SDL_OpenUrl and otherwise use the previous behavior.

File: vita3k/gui/src/app_context_menu.cpp [hunk 1]
```diff
@@ -256,17 +257,22 @@ void delete_app(GuiState &gui, EmuEnvState &emuenv, const std::string &app_path)
 }
 
 void open_path(const std::string &path) {
+    std::string s = "http";
+    if (path.size() >= s.size() && std::equal(std::begin(s), std::end(s), std::begin(path))) {
```
Comment (Zangetsu38) [inline vita3k/gui/src/app_context_menu.cpp:261]: no understund first compare here, path is obligatory more size http, no have sense

Comment (Macdu) [inline vita3k/gui/src/app_context_menu.cpp:261]:
```suggestion
    if (path.starts_with("http")) {
```

**Comment by nishinji** [comment]
Fixed.
Sorry for the trouble (I'm still a newbie).
I didn't use starts_with because the document I referred to said it was only available for C++20 or later.

**Review by Zangetsu38 — APPROVED** [review]

**Comment by Macdu** [comment]
No problem. Vita3K has C++20 as a requirement.

**Pull request merged**